src/windowsservercore/2004/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# escape=`
+FROM mcr.microsoft.com/windows/servercore:2004-amd64
+
+SHELL ["cmd", "/S", "/C"]
+USER ContainerAdministrator
+
+RUN curl -SL --output %TEMP%\python.zip https://www.nuget.org/api/v2/package/python/3.7.3 `
+    && md C:\Python C:\PythonTemp `
+    && tar -zxf %TEMP%\python.zip -C C:\PythonTemp `
+    && xcopy /s c:\PythonTemp\tools C:\Python `
+    && rd /s /q c:\PythonTemp `
+    && del /q %TEMP%\python.zip `
+    && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;"
+
+RUN md c:\\helixtmp && pushd c:\\helixtmp &&`
```

Comment:
(INCONSISTENT FORMATTING) The previous section puts `&&` at the beginning of the line where this section puts it at the end.  The most common format used in Dockerfiles is to put it at the front of the line.

Reply:
Done.